src/ApolloClient.js

```diff
@@ -0,0 +1,15 @@
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client';
+
+const cache = new InMemoryCache();
+
+const link = new HttpLink({
+  uri: 'https://cat-wiki-apollo-server.herokuapp.com',
+  cache: new InMemoryCache()
+});
```

Comment:
Why 2 separate cache objects? It's bound to end in some weird UI state error.

Reply:
Fixed